eng/pipelines/coreclr/templates/helix-queues-setup.yml

```diff
@@ -91,7 +91,6 @@ jobs:
       - ${{ if and(eq(variables['System.TeamProject'], 'public'), notIn(parameters.jobParameters.helixQueueGroup, 'pr', 'ci', 'libraries')) }}:
         - (Debian.10.Amd64.Open)Ubuntu.1804.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-10-helix-amd64-20210304164434-56c6673
         - (Debian.11.Amd64.Open)Ubuntu.1804.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-11-helix-amd64-20210304164428-5a7c380
-        - Ubuntu.1604.Amd64.Open
```

Comment:
There are more entries missing (on the next if statement).

Reply:
The next if statement is for 'internal' queue which is not running. The whole 'internal' should be removed (#57068). I will remove it for completeness.

Reply:
I thought at some point we wanted to enable internal test runs for servicing releases.